.npm/execute.js

```diff
@@ -0,0 +1,37 @@
+const spawn = require("child_process").spawn;
+
+export async function execute(binPath, args) {
+  const cca = spawn(binPath, args);
+
+  cca.on("error", error => {
+    console.error("Could not execute create-comit-app:", error);
+  });
+
+  async function handleSignal(code) {
+    cca.kill(code);
+  }
+
+  process.on("beforeExit", async code => {
```

Comment:
I am not sure if nodejs supports `async` signal handlers: https://github.com/nodejs/node/issues/29480

Reply:
Ok, I can play around with it but if it is here in the first place I think it's because I had issue without it (this is not new code).

Reply:
All of these `await` can go away anyway given [the above comment](https://github.com/comit-network/create-comit-app/pull/280/files/5d2f9126f2e8844b807752171942c169fd54564d#r356565601).